templates/views/index.pug

```diff
@@ -20,16 +20,16 @@ block content
     .the-project: .container
         .col-md-3
         .col-md-6
-            h1 The Project
-            p Forged Alliance Forever is a community-driven project designed to facilitate online play for Supreme Commander: Forged Alliance. Complete with chat, matchmaking, replay vaults, tournaments and ever evolving balance.
+            h1 What is Forged Alliance Forever?
+            p Forged Alliance Forever is a community-driven project designed to facilitate online play for Supreme Commander: Forged Alliance. Complete with chat, matchmaking, replay vaults, tournaments, Map & Mod Vault and ever-evolving balance.
```

Comment:
should we stick to the same capitalisation rules (keep lower case)?

Reply:
Sure will alter this to all lowercase